core/change.go

```diff
@@ -17,6 +19,9 @@ type ChangesConfigSorter struct {
 	config  Config
 }
 
+var errInvalidKind = errors.New("invalid kind")
+var errKindDoesNotAcceptBody = errors.New("this kind does not accept a body")
```

Comment:
```suggestion
var errKindDoesNotAcceptBody = errors.New("kind does not accept a body")
```
Just a small grammar thing, adding this feels weird to me